nextflow_schema.json

```diff
@@ -187,6 +187,157 @@
                     "hidden": true
                 }
             }
+        },
+        "clustering_params": {
+            "title": "Clustering parameters",
+            "type": "object",
+            "description": "Use these parameters to control the flow of the clustering subworkflow execution.",
+            "properties": {
+                "save_mmseqs_db": {
+                    "type": "boolean",
+                    "description": "Save the db output folder of mmseqs createdb",
+                    "help_text": "Specify to save the mmseqs formatted database of input fasta sequences."
+                },
+                "clustering_tool": {
+                    "type": "string",
+                    "description": "Choose clustering algorithm. Either simple 'cluster' for medium size inputs, or 'linclust' for less sensitive clustering of larger datasets.",
+                    "help_text": "mmseqs algorithms available: 'cluster' or 'linclust'.",
+                    "enum": ["cluster", "linclust"]
+                },
+                "cluster_seq_identity": {
+                    "type": "number",
+                    "default": 0.5,
+                    "description": "mmseqs parameter for minimum sequence identity"
```

Comment:
All of these parameters are missing more descriptive 'help_text' sections, which should better guide users when they should modify these parameters.

And no icons?! Really?! That's the best part! (Unless you did this manually andnot the schema builder?!?)

Also it's good practise to include in the longer `help_text` which parameter of the _tool_ this is modifying, if someone wants to see exactly what you are doing.

See the 'help text' section of here as an example: https://nf-co.re/taxprofiler/1.2.2/parameters/#shortread_complexityfilter_entropy